work/jira-agile.rst

```diff
@@ -47,6 +40,10 @@ External                           None (see below)
 
 The “External” team is used to label work done by individuals who are not funded by LSST.
 
+In general, every ticket should be assigned to a particular team — otherwise, nobody is responsible for ensuring it gets done.
+If you aren't sure which team to assign a ticket to, just leave the field blank, and one of the T/CAMs will pick it up.
+In the exceptional case that the ticket describes work which could be done by *either* of the Science Pipelines teams (Alert Production and Data Release Production), but it's not clear which, it can be marked with the label ``SciencePipelines`` instead of setting the team.
```

Comment:
Maybe also have a sentence like "Do not set the SciencePipelines label for tickets that have a team assigned." to make it explicit for those of us that had gotten into setting it for everything.